pkg/ddevapp/router.go

```diff
@@ -343,6 +344,27 @@ func GetRouterStatus() (string, string) {
 	return status, logOutput
 }
 
+// determineRouterHostnames returns a list of all hostnames for all active projects
+func determineRouterHostnames() []string {
+	var routerHostnames []string
+
+	for _, app := range GetActiveProjects() {
+		err := app.ReadDockerComposeYAML()
```

Comment:
I don't see a problem with this, since it only happens when we're reconfiguring the router. But it does seem like a lot of reading on some people's systems.